jms/app/pom.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.quarkus.ts.openshift</groupId>
+        <artifactId>parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../..</relativePath>
+    </parent>
+
+    <artifactId>jms-app</artifactId>
+    <packaging>jar</packaging>
+
+    <name>Quarkus OpenShift TS: JMS: Artemis Application</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-openshift</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-artemis-jms</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.activemq</groupId>
+            <artifactId>artemis-server</artifactId>
+            <version>2.10.0</version>
```

Comment:
Version should be managed in `/pom.xml`.

Reply:
`2.12.0` is the latest available

Reply:
@rsvoboda  Well, 2.12.0 throws 
```
java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
Caused by: java.lang.reflect.InvocationTargetException
Caused by: java.lang.NoClassDefFoundError: org.apache.activemq.artemis.api.core.QueueConfiguration
Caused by: java.lang.ClassNotFoundException: org.apache.activemq.artemis.api.core.QueueConfiguration
```

while 2.10.0 just works....so...... leaving 2.10.0 there. Unless you know what is the problem and how to fix it.

Reply:
ok